include/TrustWalletCore/TWEthereumAbiValueEncode.h

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TWBase.h"
+#include "TWData.h"
+#include "TWString.h"
+
+TW_EXTERN_C_BEGIN
+
+TW_EXPORT_CLASS
+struct TWEthereumAbiValueEncode;
```

Comment:
can we change the name to below?
```suggestion
struct TWEthereumAbiValue;
```

so in swift we will have
```swift
EthereumAbiValue.encodeInt32(value: 69)
// old EthereumAbiValueEncode.int32(value: 69)
```

Reply:
Best would be EthereumAbiValueEncoder.encodeInt32(), but it has encode twice.
If I make as you suggest, file name also should be EthereumAbiValue.h.

Reply:
I think I'll go with the long, full version EthereumAbiValueEncoder.encodeInt32().